.github/workflows/clang.yml

```
@@ -0,0 +1,148 @@
+      - name: 'Raise Priority for apt.postgresql.org'
+        run: |
+          cat << EOF >> ./pgdg.pref
+          Package: *
+          Pin: release o=apt.postgresql.org
+          Pin-Priority: 600
+          EOF
+          sudo mv ./pgdg.pref /etc/apt/preferences.d/
+          sudo apt update
+
+      - name: Add PostgreSQL APT repository
+        run: |
```

actionlint (1.7.7)

property "psql" is not defined in object type {release: string}

(expression)